app/controllers/reports_controller.rb

```diff
@@ -4,7 +4,25 @@ class ReportsController < ApplicationController
   before_action :verify_is_admin
 
   def index
-    @audits = Audited::Audit.all.order(created_at: :desc)
+  end
+
+  def audit_logs
+    length = params[:length].to_i
+    page = params[:start].to_i == 0 ? 1 : (params[:start].to_i / length) + 1
+    draw = params[:draw].to_i
+    search = params[:search][:value] unless params[:search].nil?
+    records = Audited::Audit.all
```

Comment:
Nit picky, but `records` might be confusing here because it's what we use in the `death_records_controller` and other places. What about `audits` or something similar?